app/controllers/api/stateless/braintree/subscriptions_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Api
+  module Stateless
+    module Braintree
+      class SubscriptionsController < StatelessController
+        before_filter :authenticate_request!
+
+        def index
+          @subscriptions = subscriptions_for_member(@current_member)
```

Comment:
Good method name. What about extracting out the AR logic into a service class - I created a poorly named class for the payment methods controller - https://github.com/SumOfUs/Champaign/blob/35c66abb8146d7ae3fd997379b97cfab890f032d/app/controllers/api/stateless/braintree/payment_methods_controller.rb#L26

It'll keep the controller class more focussed and less coupled. What do you think?